src/css/corner-layout.scss

```diff
@@ -0,0 +1,100 @@
+// Base styles
+html.easy-popup-open {
+  &:has(.easy-popup-master-corner.open) {
+    .easy-popup-master {
+      opacity: 0;
+      visibility: hidden;
+    }
+  }
+}
+
+.easy-popup-master-corner {
+  transition: opacity 0.4s ease, transform 0.4s ease;
+  transform: translateX(0);
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  max-width: 450px;
+  opacity: 1;
+  visibility: visible;
+  top: unset;
+  left: unset;
+  height: unset;
+  width: unset;
+  display: block;
+
+  &:not(.open) {
+    pointer-events: none;
+  }
+
+  .ep-close-button {
+    position: fixed;
+    transform: translate(0);
+    --ep-close-color: #000;
```

Comment:
Why do you override this variable here? CSS override should use the same selector when possible.